cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/CloudStack.java

```diff
@@ -33,8 +33,11 @@
 
     private final Optional<SpiFileSystem> fileSystem;
 
+    private final boolean createCloudWatchAlarm;
```

Comment:
could you drop this field in favor of adding only an entry to the parameter map? As it is very spi specific I think we should not include this as a field in `CloudStack`

Reply:
fixed